src/test/java/org/verdictdb/coordinator/MySqlShowTableTest.java

```diff
@@ -0,0 +1,126 @@
+package org.verdictdb.coordinator;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.verdictdb.VerdictContext;
+import org.verdictdb.commons.DatabaseConnectionHelpers;
+import org.verdictdb.connection.CachedDbmsConnection;
+import org.verdictdb.connection.DbmsConnection;
+import org.verdictdb.connection.JdbcConnection;
+import org.verdictdb.exception.VerdictDBDbmsException;
+import org.verdictdb.exception.VerdictDBException;
+import org.verdictdb.sqlsyntax.MysqlSyntax;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static org.junit.Assert.assertEquals;
+
+public class MySqlShowTableTest {
```

Comment:
Please change the name of this class to "MetaDataStatementsTest", and include the tests for other databases. Please refer to this file for avoiding writing the same code: https://github.com/mozafari/verdictdb/blob/master/src/test/java/org/verdictdb/jdbc41/JdbcQueryDataTypeTestForAllDatabases.java

Reply:
For Postgresql and Redshift, these two databases do not use show or desribe, should I test them? Currently, what ExecutionContext is doing is getting list of data from DbmsConnection.

Reply:
We should always test them, but the question is just how to test. I think the following is enough.

1. Create a random schema.
1. Create a table in the schema; the table contains two columns (one string type and one integer type).
1. Asserts that the results of our queries are the table name and the column names, respectively.